index.dd

```diff
@@ -6,7 +6,7 @@ $(SECTION1 ,
 
 $(DIVC intro, $(DIV, $(DIV,
     $(DIV,
-        $(TC p, pitch, $(B D) is a systems programming language with C-like
+        $(TC p, pitch, $(B D) is a general-purpose programming language with C-like
         syntax and static typing. It combines efficiency, control and modeling
```

Comment:
Let's do this:

D is a general-purpose language with static typing, systems-level access, and C-like syntax.